### DISCUSSION TITLE
Copilot stopped working a couple days ago and still not working JetBrains rider plugin.

### DISCUSSION THREAD

**Discussion by jajern** [community/community#40175] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Copilot on MacOS as Jetbrains Rider plugin stopped working a couple days ago and no longer works.

When it stopped working I checked the status on Githubstatus website and it showed as offline/errors.

The status now shows OK, but the plugin is still not working for me, it asks me to log in, and when I try to log in, nothing happens, the login window flashes for a second and looks like a status bar is loading something, then it disappears. I checked behind the rider IDE to make sure the login window was not being hidden behind the IDE but that was not the case.

 I have 34 days left in my trial period so it is not a billing issue. Is anyone else having trouble logging in and are there any suggestions to get it working again?

**Comment by jajern** [comment, marked as answer]
RESOLVED - In case anyone else has this issue I was able to get it working again by:
1. Uninstalling the Copilot plugin, then restart Rider IDE (as the IDE recommends)
2. Reinstall Copilot plugin, then restart the Rider IDE (as the IDE recommends)
3. Logging out of GitHub in the copilot settings, then login again

**Comment by onlyonedats** [comment]
If anyone having issues github login issues, find the solution.

Remove the latest version of VS Code from Macbook

Install an older version (1.93.1)

Update it later through the app itself: its works well.
.....

find the URL to install through terminal.

curl -L "https://update.code.visualstudio.com/1.93.1/darwin-universal/stable" -o VSCode-darwin.zip &&
unzip VSCode-darwin.zip -d /Applications

Enjoy 😊 😉
